internal/website/content/howto/pubsub/publish.md

```diff
@@ -13,34 +13,59 @@ Publishing a message to a topic with the Go CDK takes two steps:
 [Open the topic]: {{< ref "#opening" >}}
 [Send messages]: {{< ref "#sending" >}}
 
+<!--more-->
+
 ## Opening a Topic {#opening}
 
 The easiest way to open a topic is using [`pubsub.OpenTopic`][] and a URL
 pointing to the topic, making sure you ["blank import"][] the driver package to
 link it in. See [Concepts: URLs][] for more details. If you need fine-grained
 control over the connection settings, you can call the constructor function in
-the driver package directly (like `gcppubsub.OpenTopic`). [This guide][] will
-show how to use both forms for each pub/sub provider.
+the driver package directly (like `gcppubsub.OpenTopic`).
+See the [guide below][] for usage of both forms for each supported provider.
 
-[This guide]: {{< ref "#services" >}}
+[guide below]: {{< ref "#services" >}}
 [`pubsub.OpenTopic`]:
 https://godoc.org/gocloud.dev/pubsub#OpenTopic
 ["blank import"]: https://golang.org/doc/effective_go.html#blank_import
 [Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
 
-## Sending messages on a Topic {#sending}
+## Sending Messages on a Topic {#sending}
 
-Sending a message on a Topic looks like this:
+Sending a message on a [Topic](https://godoc.org/gocloud.dev/pubsub#Topic) looks
+like this:
 
 {{< goexample src="gocloud.dev/pubsub.ExampleTopic_Send" imports="0" >}}
 
-Note that the [semantics of message delivery][] can vary by provider.
+Note that the [semantics of message delivery][] can vary by backing service.
 
 [semantics of message delivery]: https://godoc.org/gocloud.dev/pubsub#hdr-At_most_once_and_At_least_once_Delivery
 
 ## Supported Pub/Sub Services {#services}
 
-### AWS Simple Notification Service {#sns}
+### Google Cloud Pub/Sub {#gcp}
+
+The Go CDK can publish to a Google [Cloud Pub/Sub][] topic. The URLs use the
+project ID and the topic ID. `pubsub.OpenTopic` will use [Application Default
+Credentials][GCP creds].
+
+{{< goexample "gocloud.dev/pubsub/gcppubsub.Example_openTopicFromURL" >}}
+
+[Cloud Pub/Sub]: https://cloud.google.com/pubsub/docs/
+[GCP creds]: https://cloud.google.com/docs/authentication/production
+
+#### GCP Cloud Pub/Sub Constructor {#gcp-ctor}
```

Comment:
I think you forgot to change this GCP back.

Reply:
Done